command/v6/enable_service_access_command_test.go

```diff
@@ -0,0 +1,246 @@
+package v6_test
+
+import (
+	"errors"
+
+	"code.cloudfoundry.org/cli/actor/actionerror"
+	"code.cloudfoundry.org/cli/actor/v2action"
+	"code.cloudfoundry.org/cli/command/commandfakes"
+	"code.cloudfoundry.org/cli/command/flag"
+	"code.cloudfoundry.org/cli/command/translatableerror"
+	. "code.cloudfoundry.org/cli/command/v6"
+	"code.cloudfoundry.org/cli/command/v6/v6fakes"
+	"code.cloudfoundry.org/cli/util/configv3"
+	"code.cloudfoundry.org/cli/util/ui"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+)
+
+var _ = Describe("enable-service-access command", func() {
+	var (
+		cmd             EnableServiceAccessCommand
+		testUI          *ui.UI
+		fakeConfig      *commandfakes.FakeConfig
+		fakeSharedActor *commandfakes.FakeSharedActor
+		fakeActor       *v6fakes.FakeEnableServiceAccessActor
+		binaryName      string
+		executeErr      error
+		extraArgs       []string
+	)
+
+	BeforeEach(func() {
+		testUI = ui.NewTestUI(nil, NewBuffer(), NewBuffer())
+		fakeConfig = new(commandfakes.FakeConfig)
+		fakeSharedActor = new(commandfakes.FakeSharedActor)
+		fakeActor = new(v6fakes.FakeEnableServiceAccessActor)
+		extraArgs = nil
+
+		cmd = EnableServiceAccessCommand{
+			UI:          testUI,
+			Config:      fakeConfig,
+			SharedActor: fakeSharedActor,
+			Actor:       fakeActor,
+			RequiredArgs: flag.Service{
+				Service: "some-service",
+			},
+		}
+
+		binaryName = "faceman"
+		fakeConfig.BinaryNameReturns(binaryName)
+	})
+
+	JustBeforeEach(func() {
+		executeErr = cmd.Execute(extraArgs)
+	})
+
+	When("the user provides arguments", func() {
+		BeforeEach(func() {
+			extraArgs = []string{"some-extra-arg"}
+		})
+
+		It("fails with a TooManyArgumentsError", func() {
+			Expect(executeErr).To(MatchError(translatableerror.TooManyArgumentsError{
+				ExtraArgument: "some-extra-arg",
+			}))
+		})
+	})
+	When("a cloud controller API endpoint is set", func() {
```

Comment:
Nit (feel free to ignore): missing newline

Reply:
Can we add lint rule for that?